wp-content/themes/cpipr/series-single.php

```diff
@@ -156,23 +159,22 @@ function getNextPost($post, $series_map)
                     <p class="date-text"><?php the_time('j F, Y')?></p>
                   </div>
                 </div>
-              </div>
+							</div>	
```

Comment:
What happened to the indentation on this line?

Reply:
we are now showing the whole block only if $next_post is not empty